homeassistant/components/local_ip/sensor.py

```diff
@@ -0,0 +1,43 @@
+"""Sensor platform for local_ip."""
+from homeassistant.const import CONF_NAME, STATE_UNKNOWN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import get_local_ip
+
+from . import DOMAIN
+
+
+async def async_setup_platform(
+    hass: HomeAssistant, config, async_add_entities, discovery_info=None
+):
+    """Set up the sensor platform."""
+    name = hass.data[DOMAIN][CONF_NAME]
+    async_add_entities([IPSensor(name)], True)
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry, async_add_entities):
+    """Set up the platform from config_entry."""
+    return await async_setup_platform(hass, {}, async_add_entities, None)
+
+
+class IPSensor(Entity):
+    """A simple sensor."""
+
+    def __init__(self, name: str):
+        """Initialize the sensor."""
+        self._state = STATE_UNKNOWN
```

Comment:
Use `None` to represent unknown state.

Reply:
Sure, but just for general knowledge, what's the point of the const that I used?

Reply:
It's used by the base entity class. There `None` is converted to `STATE_UNKNOWN`.